utils/exec-shell-command.js

```diff
@@ -0,0 +1,13 @@
+function execShellCommand(cmd) {
+ const exec = require('child_process').exec;
```

Comment:
Why the require inside the body of this function? I assume that the module will be cached so subsequent calls to `execShellCommand` won't result in a fresh require, but unsure what benefit there might be to having it here.

Reply:
yeah that won't result in `fresh require`. The only benefit is during server initialization the require call won't be processed and will only be processed once we actually need it. But I guess to follow common pattern I would probably add it outside.